config/initializers/cookie_rotator.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
-# As part of the Rails 7 upgrade, we need to convert legacy (SHA1) cookies to SHA256.
-# This can be removed after it has been in production for a little bit.
+# Due to the rolled back deploy, some users have SHA256 cookies.
+# Read them, but write back SHA1 cookies (writing is based on current setting of config.active_support.key_generator_hash_digest_class)
```

Comment:
Since we ended up not pushing this out, could you update the comment to something along the lines of
```suggestion
# To support rolling deploys, force all session cookies to be compatible with older versions.
# Read the cookies, but write back SHA1 cookies (writing is based on current setting of config.active_support.key_generator_hash_digest_class)
```

Reply:
I took this suggestion and added the steps we need to take in the future and a short explanation.